src/components/NcGuestContent/NcGuestContent.vue

```diff
@@ -49,6 +52,12 @@ This components provides a wrapper around a guest page content.
  */
 export default {
 	name: 'NcGuestContent',
+	mounted() {
+		document.getElementById('content').classList.add('guest')
```

Comment:
nit: let's use something more indicative that the class comes from a nextcloud/vue component. e.g. vue-guest or something. otherwise someone could wrongly assume that the `guest` is added by Nextcloud and base some other styles on it

we could even use a random identifier here, if technically possible. like we did in https://github.com/nextcloud/nextcloud-axios/pull/510#discussion_r980902444

Reply:
Done. I used the class '.nc-guest-content'.